python/kserve/kserve/storage/__init__.py

```diff
@@ -11,7 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 # flake8: noqa
 
-from kserve.storage.storage import Storage
+# Keep backwards capability, try import storage package.
+# This way, existing projects using the Python SDK will not break, allowing
+# users to upgrade the SDK without changing their code.
```

Comment:
Can you test the backwards compatibility?

Reply:
I've tested manually, let me try to add a test case.

Reply:
added a test.

Reply:
https://github.com/kserve/kserve/actions/runs/12678720187/job/35336887961?pr=3902

Reply:
@sivanantha321 ^